src/QRCoders.jl

```diff
@@ -3,30 +3,34 @@ Module that can create QR codes as data or images using `qrcode` or `exportqrcod
 """
 module QRCoders
 
-# create QR code
-export qrcode, exportqrcode, exportbitmat, QRCode
-
-# supported modes
-export Mode, Numeric, Alphanumeric, Byte, Kanji, UTF8
-
-# error correction levels
-export ErrCorrLevel, Low, Medium, Quartile, High
+export
+    # create QR code
+    qrcode, exportqrcode, exportbitmat, QRCode,
+    
+    # supported modes
+    Mode, Numeric, Alphanumeric, Byte, Kanji, UTF8,
+    
+    # error correction levels
+    ErrCorrLevel, Low, Medium, Quartile, High,
 
-# get information about QR code
-export getmode, getversion, qrwidth
+    # get information about QR code
+    getmode, getversion, qrwidth, getsegments,
 
-# data type of Reed Solomon code
-export Poly, geterrcode
+    # data type of Reed Solomon code
+    Poly, geterrcode,
 
-# error type
-export EncodeError
+    # error type
+    EncodeError,
 
-# QR code style
-export unicodeplot, unicodeplotbychar
+    # QR code style
+    unicodeplot, unicodeplotbychar,
+    imageinqrcode, animatebyqrcode
 
 using ImageCore
 using FileIO
 using UnicodePlots
+using ImageTransformations
+using StatsBase: sample
```

Comment:
Also, seems not used here?

Reply:
Yes, `StatsBase` should be removed, since `sample` is replaced by a more feasible method `pickcodewords` in the current commit.

---

I focus more on the effect rather than efficiency in this PR, so there may be more to optimize. I'll squash them after some performance test.

Reply:
Same as the other two. I failed to see where it is used by looking at the diffs.